2021q2/doceng.adoc

```diff
@@ -0,0 +1,54 @@
+=== Documentation Engineering Team
+
+Link: link:https://www.freebsd.org/docproj/[FreeBSD Documentation Project] +
+Link: link:https://docs.freebsd.org/en/books/fdp-primer/[FreeBSD Documentation Project Primer for New Contributors] +
+Link: link:https://www.freebsd.org/administration/#t-doceng[Documentation Engineering Team]
+
+Contact: FreeBSD Doceng Team <doceng@FreeBSD.org> +
+
+The doceng@ team is a body to handle some of the meta-project issues associated with the FreeBSD Documentation Project, more infomation on link:https://www.freebsd.org/internal/doceng/[FreeBSD Doceng Team Charter].
```

Comment:
s/body to handle/group handling/
s/infomation on/information in the/
Also, I'd change the "," before "more" to ";", but that may be a matter of taste.

Reply:
This text is part of  the [FreeBSD Doceng Team Charter](https://www.freebsd.org/internal/doceng/).